packages/webcomponents/src/components/business-forms/owner-form/identity-address/identity-address-form.tsx

```diff
@@ -66,16 +64,29 @@ export class BusinessAddressForm {
               inputHandler={(name, value) => this.inputHandler(name, value)}
             />
           </div>
-
           <div class="col-12 col-md-6">
-            <form-control-number
+            <form-control-text
               name="postal_code"
               label="Postal Code"
               defaultValue={this.defaultValues?.postal_code}
               error={this.errors?.postal_code}
+              maxLength={5}
+              keyDownHandler={(e) => filterPostalInput(e)}
               inputHandler={(name, value) => this.inputHandler(name, value)}
```

Comment:
Just a small improvement: updating the assignment of the inputHandler in your component to use a direct reference to the method, like this:

```
inputHandler={this.inputHandler}
```

Benefits of this approach include:


Error Traceability: Using a named function improves the clarity of stack traces during debugging. It's easier to identify issues because the function name appears directly in error messages.

Performance Optimization: By referencing the handler directly, the function is not recreated on each render. This prevents unnecessary re-renders of the form-control-text component.

Code Readability and Inspection: While using direct method references might seem less explicit about what arguments are passed, most development environments allow you to hover over the method name to see its signature, thus maintaining an easy way to understand the expected arguments.

Reply:
Great call - thank you!